src/repository/interface.rs

```diff
@@ -295,7 +310,7 @@ pub mod tests {
         // Test loading all columns
```

Comment:
Add a test here for a specific table version to make sure the query is correct for PG and SQLite?